File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/common/utils/ToastUtils.java
package com.trtc.uikit.livekit.common.utils;

import android.content.Context;

import androidx.annotation.StringRes;

import com.tencent.qcloud.tuicore.TUIConfig;
import com.tencent.qcloud.tuicore.util.ToastUtil;

public class ToastUtils {
    public static void toast(@StringRes int resourceId) {
        Context context = TUIConfig.getAppContext();
        if (context == null) {
            return;
        }
        if (resourceId == 0) {
            return;
        }
        ToastUtil.toastShortMessage(context.getString(resourceId));
    }
}

File: Android/ktv/src/main/java/com/tencent/cloud/uikit/livekit/utils/ToastUtils.kt
package com.tencent.cloud.uikit.livekit.utils

import android.content.Context
import com.trtc.tuikit.common.util.ToastUtil

/**
 * This is a packaging class that can be replaced by internal implementations.
 * Prepare for future replacement implementation.
 */
object ToastUtils {
    fun showToast(context: Context, text: String) {
        ToastUtil.toastShortMessage(text)
    }
}